docs/management/release.rst

```diff
@@ -15,6 +15,13 @@ not include an update to the catalog on Gadi, skip the first two steps below:
       $ export RELEASE=vX.X.X
       $ cd bin
       $ qsub -v version=${RELEASE} build_all.sh
+
+   #. If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To workaround::
```

Comment:
```suggestion
   #. If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To download and cache the schema, first import the `access_nri_intake.source` and `access_nri_intake.catalog` sub-packages from a Gadi node with network access (e.g. a login or ARE node). I.e., using the release version of `access_nri_intake`::
```

Reply:
Also, this could go in a `.. note::` block - see [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes)